src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -189,6 +193,67 @@ public async Task ExecuteAsync(string stackName, bool saveCdkProject)
             await orchestrator.DeployRecommendation(cloudApplication, selectedRecommendation);
         }
 
+        private void ConfigureDeployment(Recommendation recommendation, IEnumerable<OptionSettingItem> configurableOptionSettings, UserDeploymentSettings userDeploymentSettings)

```

Comment:
Can you add a test case? Also having a sample input JSON file would give better idea how it is consumed.